src/views/components/pyi-escape-m2b-form.njk

```diff
@@ -1,50 +0,0 @@
-{% from "govuk/components/radios/macro.njk" import govukRadios %}
```

Comment:
Was this used anywhere before this PR? I can't see where it was used

Reply:
On line 45

Reply:
Sorry, I meant the now-deleted file/ pyi-escape-m2b-form

Reply:
Not that I could see, that's why I deleted it :)